js/scenes/Growers/Growers.js

```diff
@@ -1,23 +1,44 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { View, Text } from 'react-native';
+import { ScrollView, View, Text, Image, TouchableOpacity } from 'react-native';
 import { styles } from './styles';
+import LineSeparator from '../../components/LineSeparator/';
+import { colors } from '../../config/styles';
 
-const Growers = ({ growersList }) => (
-    <View style={styles.container}>
-        <Text>This is Growers scene</Text>
-    </View>
-);
+const Growers = ({ growersList, growerLink }) => (
+    <ScrollView>
+        <View style={styles.container}>
+        {
+            growersList.map((grower) => (
+                <View key={grower.id} style={styles.mainContainer}>
```

Comment:
It'd be nice to separate the concerns and abstract this inner component (grower).

Reply:
Changes has been made regarding this issue. Its ready for your review.
Thanks